haiku-apps/bedc/bedc-0.5.0~git.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="A P2P file sharing client for the Direct Connect Protocol"
+DESCRIPTION="BeDC is a BeOS file sharing client compatible \
+with the Direct Connect protocol.
+Currently only chatting works, not filesharing/downloading."
+HOMEPAGE="https://github.com/HaikuArchives/BeDC"
+COPYRIGHT="2002 The BeDC team"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+srcGitRev="cf2f67fba574e934a06026c66b4f4298470716c8"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9a4a632fa21cfd7d1b9adfca3179356d81e26308d67a0e6dcbfdd4546bb2d18c"
+SOURCE_FILENAME="BeDC-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="BeDC-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	BeDC = $portVersion
+	app:BeDC = $portVersion
+	"
```

Comment:
```diff
 PROVIDES="
 -	BeDC = $portVersion
+	bedc = $portVersion
 	app:BeDC = $portVersion
 	"
```
BTW, I don't know if we should also **_add_** a `app:BDirectConnect` (or, maybe, **_replace_** `app:BeDC` by `app:BDirectConnect`.)

@pulkomandy, the runtime in `$appsDir` is called BDirectConnect, but the desktop symlink is called BeDC. Do you know if we should have an `app:` for both of these names or for just one of them? Thanks!
(These situations are not oommon, and doing metrics here wouldn't make any sense :)

Reply:
Does anyone object in renaming the recipe `bdirectconnect-0.5.0~git.recipe` (and moving it to `haiku-apps/bdirectconnect/`) ?

I think BeDC is too short. Moreover, the runtime is called BDirectConnect.

We would have:
```
PROVIDES="
	bdirectconnect = $portVersion
	app:BDirectConnect = $portVersion
	"
```
```
defineDebugInfoPackage bdirectconnect \
	$appsDir/BDirectConnect
```
```
INSTALL()
{
	install -d $appsDir
	install -t $appsDir objects/BDirectConnect

	addAppDeskbarSymlink $appsDir/BDirectConnect
}
```

And no more questions about what `app:` we should provide :)